lib/cogctl/actions/relays/create.ex

```diff
@@ -0,0 +1,33 @@
+defmodule Cogctl.Actions.Relays.Create do
+  use Cogctl.Action, "relays create"
+  import Cogctl.Actions.Relays.Util, only: [render: 3]
+
+  def option_spec do
+    [{:name, :undefined, 'name', {:string, :undefined}, 'Relay name (required)'},
```

Comment:
Lose the `'name'` here so we can do `cogctl relays create <name> --token <token> --description <desc>`. I want to move away from `--name` everywhere on this type of create.